ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -37,6 +51,38 @@ export default function EditGasPopover({
     }
   }, [showSidebar, onClose, dispatch]);
 
+  const onSubmit = useCallback(() => {
+    if (transaction && mode) {
```

Comment:
To avoid one level of nesting, we can do an early return:
```
if (!transaction || !mode) {
  return;
}
```

Reply:
Done in latest